i18n/ko.json

```
@@ -1,1962 +1,1962 @@
-    "transcoding_reference_frames": "Reference frames",
-    "transcoding_reference_frames_description": "The number of frames to reference when compressing a given frame. Higher values improve compression efficiency, but slow down encoding. 0 sets this value automatically.",
-    "transcoding_required_description": "Only videos not in an accepted format",
-    "transcoding_settings": "Video Transcoding Settings",
-    "transcoding_settings_description": "Manage which videos to transcode and how to process them",
-    "transcoding_target_resolution": "Target resolution",
-    "transcoding_target_resolution_description": "Higher resolutions can preserve more detail but take longer to encode, have larger file sizes, and can reduce app responsiveness.",
+    "transcoding_accepted_audio_codecs": "허용 오디오 코덱",
+    "transcoding_accepted_audio_codecs_description": "트랜스코딩이 불필요한 오디오 코덱을 선택하세요. 특정 트랜스코딩 정책에서만 사용됩니다.",
+    "transcoding_accepted_containers": "허용 컨테이너",
+    "transcoding_accepted_containers_description": "MP4로 리멕싱이 불필요한 컨테이너 포맷을 선택하세요. 특정 트랜스코딩 정책에서만 사용됩니다.",
+    "transcoding_accepted_video_codecs": "허용 비디오 코덱",
+    "transcoding_accepted_video_codecs_description": "트랜스코딩이 불필요한 비디오 코덱을 선택하세요. 특정 트랜스코딩 정책에서만 사용됩니다.",
+    "transcoding_advanced_options_description": "대부분의 사용자에게는 변경이 필요 없는 고급 옵션입니다.",
+    "transcoding_audio_codec": "오디오 코덱",
+    "transcoding_audio_codec_description": "Opus는 가장 뛰어난 품질을 제공하지만, 구형 장치나 소프트웨어와의 호환성이 낮을 수 있습니다.",
+    "transcoding_bitrate_description": "최대 비트레이트를 초과하거나 허용 포맷이 아닌 영상",
+    "transcoding_codecs_learn_more": "용어에 대한 자세한 설명은 FFmpeg 문서를 참조하세요: <h264-link>H.264</h264-link>, <hevc-link>HEVC</hevc-link>, <vp9-link>VP9</vp9-link>",
+    "transcoding_constant_quality_mode": "고정 품질 모드",
+    "transcoding_constant_quality_mode_description": "ICQ가 CQP보다 우수하지만 일부 하드웨어는 지원하지 않습니다. 이 옵션을 설정하면 품질 기반 인코딩 시 해당 모드를 우선 사용합니다. NVENC는 ICQ를 지원하지 않기 때문에 무시됩니다.",
+    "transcoding_constant_rate_factor": "고정 비트율 계수 (-crf)",
+    "transcoding_constant_rate_factor_description": "영상 품질 수준입니다. 일반값: H.264=23, HEVC=28, VP9=31, AV1=35. 값이 낮을수록 품질은 좋지만 파일이 커집니다.",
+    "transcoding_disabled_description": "모든 비디오 트랜스코딩 비활성화 (일부 클라이언트에서 재생 오류 발생 가능)",
+    "transcoding_encoding_options": "인코딩 옵션",
+    "transcoding_encoding_options_description": "인코딩에 사용할 코덱, 해상도, 품질 등을 설정합니다.",
+    "transcoding_hardware_acceleration": "하드웨어 가속",
+    "transcoding_hardware_acceleration_description": "실험적 기능입니다. 속도는 빠르지만 동일 비트레이트 기준으로 품질이 낮을 수 있습니다.",
+    "transcoding_hardware_decoding": "하드웨어 디코딩",
+    "transcoding_hardware_decoding_setting_description": "인코딩뿐 아니라 디코딩도 하드웨어 가속을 사용합니다. 일부 영상에는 작동하지 않을 수 있습니다.",
+    "transcoding_hevc_codec": "HEVC 코덱",
+    "transcoding_max_b_frames": "최대 B-프레임 수",
+    "transcoding_max_b_frames_description": "값이 높을수록 압축 효율은 좋아지지만 인코딩 속도는 느려집니다. 구형 장치에서는 호환되지 않을 수 있습니다. 0은 비활성화, -1은 자동 설정",
```

```suggestion
    "transcoding_max_b_frames_description": "값이 높을수록 압축 효율은 좋아지지만 인코딩 속도는 느려집니다. 구형 장치에서는 호환되지 않을 수 있습니다. 0은 비활성화, -1은 자동 설정",
```
I think keeping the original translation sounds more natural. If the goal is to make the sentence more concise, it would be cleaner to enclose example settings in parentheses consistently. (Some examples are missing periods, which affects consistency.)